### DISCUSSION TITLE
bug in AraGPT2 colab

### DISCUSSION THREAD

**Discussion by srulikbd** [aub-mind/arabert#57] [Q&A] [answered]
hey. thanks for your amazing code and time!!
there is some issue with the code. maybe need to update the specific packages versions that you used..
I'm trying run the colab example of GPT and got the following error (also locally) : 
```
/content/arabert/aragpt2/grover/modeling_gpt2.py in forward(self, input_ids, past_key_values, attention_mask, token_type_ids, position_ids, head_mask, inputs_embeds, encoder_hidden_states, encoder_attention_mask, labels, use_cache, output_attentions, output_hidden_states, return_dict)
    915             hidden_states=transformer_outputs.hidden_states,
    916             attentions=transformer_outputs.attentions,
--> 917             cross_attentions=transformer_outputs.cross_attentions,
    918         )
    919 

TypeError: __init__() got an unexpected keyword argument 'past_key_values'
```

**Comment by WissamAntoun** [comment, marked as answer]
which transformers version are you using. It should be 4.2.1 the latest

**Reply by srulikbd** [reply]
great, it works!
so we just need to change it in the colab example.
thanks.

**Reply by WissamAntoun** [reply]
> great, it works!
> so we just need to change it in the colab example.
> thanks.

done
